src/Appwrite/Platform/Workers/Migrations.php

```diff
@@ -291,21 +294,26 @@ protected function processMigration(Document $project, Document $migration, Log
             if (!empty($sourceErrors) || !empty($destinationErrors)) {
                 $migrationDocument->setAttribute('status', 'failed');
                 $migrationDocument->setAttribute('stage', 'finished');
+                $log->addTag('stage', 'finished');
 
-                $errorMessages = [];
+                $prettyMessages = [];
+                $simpleMessages = [];
```

Comment:
why do we need two variables here? I see they contain the same information. resourceType, resourceId & message